docs/Documentation-CHN/UserGuide/5-Operation Manual/2-DML (Data Manipulation Language).md

```diff
@@ -147,13 +147,13 @@ select wf01.wt01.status,wf02.wt02.hardware from root.ln where (time > 2017-11-01
 
 ### 降频聚合查询
 
-本章节主要介绍降频聚合查询的相关示例，主要使用的是IoTDB SELECT语句的[GROUP BY子句](/#/Documents/progress/chap5/sec4)，该子句是IoTDB中用于根据用户给定划分条件对结果集进行划分，并对已划分的结果集进行聚合计算的语句。IoTDB支持根据时间间隔对结果集进行划分，默认结果按照时间升序排列。同时，您也可以使用Java JDBC标准接口来执行相关的查询语句。
+本章节主要介绍降频聚合查询的相关示例，主要使用的是IoTDB SELECT语句的[GROUP BY子句](/#/Documents/progress/chap5/sec4)，该子句是IoTDB中用于根据用户给定划分条件对结果集进行划分，并对已划分的结果集进行聚合计算的语句。IoTDB支持根据时间间隔和自定义的滑动步长（默认值与时间间隔相同，自定义的值必须大于等于时间间隔）对结果集进行划分，默认结果按照时间升序排列。同时，您也可以使用Java JDBC标准接口来执行相关的查询语句。
 
 GROUP BY语句为用户提供三类指定参数：
 
-* 参数1：划分时间轴的时间间隔参数
-* 参数2：时间轴划分原点参数（可选参数）
-* 参数3：时间轴显示时间窗参数（一个或多个）
+* 参数1：时间轴显示时间窗参数
+* 参数2：划分时间轴的时间间隔参数
+* 参数3：滑动步长（可选参数）
```

Comment:
Besides the constraint that the third parameter should not be smaller than the second parameter, are there other constraints of the three parameters? For example, should the second and the third parameters be positive integers?

Reply:
And I tested the sqls
- `select count(*) from root.ln.wf01.wt01 group by([1509465600000,1509465600000],0m)`
- `select count(*) from root.ln.wf01.wt01 group by([1509465600000,1509466140000],5s,0m)`

both get
```
Msg: / by zero
```
I tested the sql 
- `select count(*) from root.ln.wf01.wt01 group by([1509465600000,1509466140000],0s,1m)`

The result is 
```
+-----------------------------------+------------------------------------+------------------------------------+
|                               Time|     count(root.ln.wf01.wt01.status)|count(root.ln.wf01.wt01.temperature)|
+-----------------------------------+------------------------------------+------------------------------------+
|      2017-11-01T00:00:00.000+08:00|                                   0|                                   0|
|      2017-11-01T00:01:00.000+08:00|                                   0|                                   0|
|      2017-11-01T00:02:00.000+08:00|                                   0|                                   0|
|      2017-11-01T00:03:00.000+08:00|                                   0|                                   0|
|      2017-11-01T00:04:00.000+08:00|                                   0|                                   0|
|      2017-11-01T00:05:00.000+08:00|                                   0|                                   0|
|      2017-11-01T00:06:00.000+08:00|                                   0|                                   0|
|      2017-11-01T00:07:00.000+08:00|                                   0|                                   0|
|      2017-11-01T00:08:00.000+08:00|                                   0|                                   0|
|      2017-11-01T00:09:00.000+08:00|                                   0|                                   0|
+-----------------------------------+------------------------------------+------------------------------------+
Total line number = 10
It costs 0.037s
```

Reply:
Both should be positive. Thanks for reminding.